mantis-connectors/mantis-connector-iceberg/src/main/java/io/mantisrx/connector/iceberg/sink/committer/config/CommitterProperties.java

```diff
@@ -28,6 +28,7 @@ private CommitterProperties() {
      * Iceberg committer frequency by time in milliseconds.
      */
     public static final String COMMIT_FREQUENCY_MS = "commitFrequencyMs";
+    public static final String WATERMARK_PROPERTY_KEY = "watermarkPropertyKey";
```

Comment:
Do we ask user to set per-region config value for this key?

Reply:
We can set this at the job level.

Reply:
In Flink the three jobs in different regions sets region-specific VTTS table properties. If we want to keep the same behavior in Mantis, we may need to add `region` to the key name?